File: v3/php/api_check_ins.php
<?php
/*
 * API Version: 3.0
 * This is a basic API example that will show a list of your last 20 check-ins
 */

//YOU MUST SET YOUR API KEY IN THE CONFIG FILE FIRST
require_once 'config.php';

$api_result = simplexml_load_file("https://www.smartwaiver.com/api/" . API_VERSION . "/?rest_request=" . API_KEY . "&restapi_checkins=10");

if((!isset($api_result->checkins)) || (!isset($api_result->checkins->checkin))) {
    return false; //if you don't have any checkins it'll fail here
}

foreach($api_result->checkins->checkin as $checkin) {
    echo "Participant ID: " . $checkin->participant_id;
    echo "<br>Date UTC: " . $checkin->date_utc;
    echo "<hr>";
}

File: v3/rubyonrails/api_check_ins.rb
class ApisController < ApplicationController
   def check_ins
     checkin_limit = 10
     @api_result = Smartwaiver.get_checkins( checkin_limit )
     if @api_result["checkins"].blank? or @api_result["checkins"]["checkin"].blank?
       @output = "No Waivers found." # if you don't have any waivers it'll fail here

     else
       @output = ""

       # DOWNLOAD EACH WAIVER (You can bulk download up to 100 waivers at a time by modifying &rest_limit above to the desired # of participants)
       @api_result["checkins"]["checkin"].each do |check|
         @output << "Participant ID:" + check["participant_id"] + "<br/> Date UTC:" + check["date_utc"] + "<hr>"
       end

     end
   end
end